.github/workflows/auto-approve.yaml

```diff
@@ -0,0 +1,53 @@
+name: Approve Renovate PR
+
+on:
+  pull_request:
+    types:
+    - review_requested
+
+jobs:
+  pre-approve:
+    # Avoid running the 'auto-approve' environment if we don't need to.
+    name: Pre-Approve
+    runs-on: ubuntu-latest
+    if: ${{
+         github.event.pull_request.user.login == 'cilium-renovate[bot]' &&
+         github.triggering_actor == 'cilium-renovate[bot]' &&
+         github.event.requested_reviewer.login == 'ciliumbot'
+        }}
+    steps:
+    - name: Debug
```

Comment:
Are you going to remove Debug steps?

Reply:
No, the "steps" field is mandatory on a job so it doesn't hurt to leave them here.

Reply:
Gotcha, thanks!